client/coral-admin/src/containers/ModerationQueue/ModerationQueue.js

```diff
@@ -93,6 +98,11 @@ class ModerationQueue extends React.Component {
               className={`mdl-tabs__tab ${styles.tab}`}>{lang.t('modqueue.rejected')}</a>
             <a href='#flagged' onClick={() => this.onTabClick('flagged')}
               className={`mdl-tabs__tab ${styles.tab}`}>{lang.t('modqueue.flagged')}</a>
+            <a href='#' onClick={() => this.showShortcuts()}
```

Comment:
There's no need to wrap this `<a href='#' onClick={() => this.showShortcuts()}` 
We can just `<a href='#' onClick={this.showShortcuts}`